src/System.ComponentModel.Annotations/pkg/System.ComponentModel.Annotations.pkgproj

```diff
@@ -1,26 +1,27 @@
 ﻿<Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\System.ComponentModel.Annotations.csproj">
-      <SupportedFramework>net461</SupportedFramework>
-    </ProjectReference>
     <ProjectReference Include="..\src\System.ComponentModel.Annotations.csproj" />
     <HarvestIncludePaths Include="ref/netcore50;lib/netcore50" />
     <HarvestIncludePaths Include="ref/netstandard1.1" />
     <HarvestIncludePaths Include="ref/netstandard1.3" />
     <HarvestIncludePaths Include="ref/netstandard1.4;lib/netstandard1.4" />
+    <HarvestIncludePaths Include="ref/netstandard2.0;lib/netstandard2.0" />
+    <HarvestIncludePaths Include="ref/net461;lib/net461" />
     <InboxOnTargetFramework Include="$(AllXamarinFrameworks)" />
     <InboxOnTargetFramework Include="net45">
       <FrameworkReference>System.ComponentModel.DataAnnotations</FrameworkReference>
     </InboxOnTargetFramework>
-    <InboxOnTargetFramework Include="netcoreapp2.0" />
     <InboxOnTargetFramework Include="uap10.0.16299" />
     <InboxOnTargetFramework Include="win8" />
     <InboxOnTargetFramework Include="portable-net45+win8" />
     <ValidatePackageSuppression Include="PermitMissingImplementation">
       <!-- Supported inbox by all ns1.1-1.3 frameworks -->
       <Value>.NETStandard,Version=v1.1;.NETStandard,Version=v1.2;.NETStandard,Version=v1.3</Value>
     </ValidatePackageSuppression>
+    <ValidatePackageSuppression Include="TreatAsOutOfBox">
```

Comment:
I assuming it is ok to have two separate ValidatePackageSuppresion in the same pkgproj.